src/TuDa.CIMS.Web/Components/FilterComponent.razor

```diff
@@ -0,0 +1,73 @@
+﻿
+@* FilterComponent *@
+
+<!-- Frame -->
+<form>
+    <div class="p-3 bg-light rounded" style="width: 250px;">
+
+
+        <!-- Keyword Tags -->
+        <h6>Keywords</h6>
+        <div class="d-flex flex-wrap gap-2 mb-3">
+            @foreach (var keyword in Keywords)
+            {
+                <span class="badge bg-secondary">
+                    @keyword <button class="btn-close btn-close-white btn-sm ms-2-5" @onclick="() => RemoveKeyword(keyword)"></button>
+                </span>
+            }
+        </div>
+
+
+        <!-- Checkbox List -->
+        @foreach (var item in CheckboxItems)
+        {
+            <div class="form check mb-2">
+                <input class="form-check-input" type="checkbox" value="" id="@item.Label">
+                <label class="form-check-label" for="@item.Label">
+                    <strong>@item.Label</strong>
+                    <div class ="small text-muted"> @item.Description</div>
+                </label>
+            </div>
+        }
+        <!-- Slider -->
+        <div class="mt-3">
+            <label for="priceSlider" class="form-label"><strong>Label</strong></label>
+            <div class="d-flex justify-content-between small">
+                <span>@MinPrice€</span>
+                <span>@MaxPrice€</span>
+            </div>
+            <input type="range" class="form-range" id="priceSlider">
+        </div>
+    </div>
+</form>
+
+@code {
+
+// Keywords for the Filter
+private List<string> Keywords { get; set; } = new List<string> { "Keyword", "Key", "Word" };
+
+private void RemoveKeyword(string keyword)
+{
+Keywords.Remove(keyword);
+}
+
+// Checkbox Item Class
+private class CheckboxItem
+{
+    public string Label { get; set; }
+    public string Description { get; set; }
+}
```

Comment:
🔧The Attributes of the class could be null at the end of the Constructor. Maybe we should require them.
```suggestion
private class CheckboxItem
{
    public require string Label { get; set; }
    public require string Description { get; set; }
}
```